backend-ci/app/Commands/TestAttach.php

```
@@ -0,0 +1,40 @@
+/**
+ * Quick manual check for media attach on real DB (no mocks).
+ *
+ * @agent-command: Debug attach images
+ */
+class TestAttach extends BaseCommand
+{
+    protected $group = 'custom';
+    protected $name = 'test:attach';
+    protected $description = 'Attach images to a hardcoded variant and dump DB rows.';
+
+    public function run(array $params)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$params'. (undefined)

(UnusedFormalParameter)